### DISCUSSION TITLE
[Feature] SEMVER Docker image tags

### DISCUSSION THREAD

**Discussion by Quadrubo** [immich-app/immich#11807] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

As of the point of writing this, `v1.112.1` is the latest Immich version.
As I'm manually updating my images and not using something like release, I have to change the `IMMICH_VERSION` variable every new release. It would be nice if I wouldn't have to do that for patch releases (minor releases too).

Currently these tags are provided:
- `v1.112.1` - Version 1.112.1
- `v1.112.0` - Version 1.112.0
- `v1.111.0` - Version 1.111.0
- `release`   - Version 1.112.1

I would propose a tag system like this:
- `v1.112.1` - Version 1.112.1
- `v1.112.0` - Version 1.112.0
- `v1.112`    - Version 1.112.1
- `v1`           - Version 1.112.1
- `release`   - Version 1.112.1

This would allow pinning to specific version and allows the user to choose which semver types they are okay with auto updating.
I for example would have pinned to `v1.112` and would have automatically gotten the hotfix to `v1.112.1

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#4149

**Reply by Quadrubo** [reply]
Ah sorry I didn't see this. Thank you :)
